cli/README.md

```diff
@@ -0,0 +1,176 @@
+<div align="center">
+    <img
+    width="100%"
+    src="https://github.com/roboflow/inference/blob/main/banner.png?raw=true"
+    />
+
+<br>
+
+[notebooks](https://github.com/roboflow/notebooks) | [inference](https://github.com/roboflow/inference) | [autodistill](https://github.com/autodistill/autodistill) | [collect](https://github.com/roboflow/roboflow-collect)
+
+<br>
+
+[![version](https://badge.fury.io/py/roboflow.svg)](https://badge.fury.io/py/inference-cli)
+[![downloads](https://img.shields.io/pypi/dm/inference-cli)](https://pypistats.org/packages/inference-cli)
+[![license](https://img.shields.io/pypi/l/inference-cli)](https://github.com/roboflow/inference/blob/main/LICENSE)
+[![python-version](https://img.shields.io/pypi/pyversions/inference-cli)](https://badge.fury.io/py/inference-cli)
+
+</div>
+
+# Roboflow Inference CLI
```

Comment:
Should we make this file more specific to the CLI? Some of the repeated items here feel a little off to me if I had just read through the main package README.

Reply:
I'll take another pass at it.